src/css/style.css

```diff
@@ -0,0 +1,224 @@
+    max-width: 50%;
+}
+
+.lobby a {
+    color: var(--orange-dark);
+    text-decoration: none;
+}
+
+.lobby a:hover:enabled {
+    text-decoration: underline;
+}
+
+
+/* In-call */
+
+.incall {
+    display: flex;
+    flex-direction: column;
+}
+
+#callContainer {
+    display: flex;
+    flex-direction: row;
+    max-width: 900px;
+}
+
+#participants {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: flex-end;
+}
+
+#controls {
+    margin-bottom: 25px;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: center;
+    padding: 0 2px 0 2px;
+}
+
+#participants .participant {
+    width: 300px;
+    height: 180px;
+    background-color: var(--teal);
+}
+
+video {
+    width: 100%;
+    height: 100%;
+}
+
+/* Chat */
+
+#chat {
+    margin-left: 14px;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    width: 200px;
+}
+
+#messages {
+    height: 250px;
+    overflow-y: scroll;
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    width: 100%;
+    border: 1px solid var(--teal-dark);
+    padding: 5px;
+    border-radius: 8px;
+    margin-bottom: 8px;
+}
+
+.chatName {
+    color: var(--orange);
+}
+
+/* Form and button styling */
+
+form {
+    margin-bottom: 10px;
+    margin-bottom: 10px;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+}
+
+input,textarea {
+    padding: 7px;
+    border-color: var(--orange);
+    width: 180px;
+    border-radius: 6px;
+    font-size: 1.2em;
+    margin-bottom: 8px;
+}
+
+#chat textarea {
+    font-size: 1em;
+    height: 40px;
+}
+
+form button {
+    font-size: 1.2em;
+    width: 100px;
+}
+
+.dark-btn {
+  background-color: var(--dark);
+  border-color: var(--dark);
+  border-radius: 30px;
+  border-width: 1px;
+  color: var(--light);
+  transition: 0.5s;
+  padding: 8px;
+}
+
+.dark-btn:hover:enabled {
+    background-color: var(--light);
+    color: var(--dark);
+    cursor: pointer;
+}
+
+.light-btn {
+    background-color: var(--light);
+    border-color: var(--dark);
+    border-radius: 30px;
+    border-width: 1px;
+    color: var(--dark);
+    transition: 0.5s;
+    padding: 8px;
+  }
+  
+  .light-btn:hover:enabled {
+      background-color: var(--dark);
+      color: var(--light);
+      cursor: pointer;
+  }
+
+  .teal-btn {
+    background-color: var(--teal);
+    border-color: var(--dark);
+    border-radius: 30px;
+    border-width: 1px;
+    color: var(--dark);
+    transition: 0.5s;
+    padding: 8px;
+  }
+
+  .teal-btn:hover:enabled {
+    background-color: var(--light);
+    cursor: pointer;
+}
+
+#incall #leave:hover:enabled {
+    background-color: var(--orange);
+}
+
+#incall input {
+    border-color: var(--teal);
+}
+
+#send:hover {
+    background-color: var(--teal);
+}
+
+#controls button {
+    margin: 0 2px 0 2px;
+}
+
+#reactions button {
+    font-size: 1.5em;
+}
+
+.reaction {
+    position: absolute;
+    font-size: 5em;
```

Comment:
This is actually on purpose! I vary the sizes and opacities a bit to enable "emoji spam" - so that multiple clicks don't just blend into the same emoji that is already up, and look distinctly separate.